test/src/UnitTests.cpp

```diff
@@ -0,0 +1,41 @@
+// Note: No need to increase compile time by including UnitTests.hpp
+#include <SFML/System/String.hpp>
+#include <SFML/System/Time.hpp>
+#include <SFML/Window/VideoMode.hpp>
+#include <SFML/Graphics/Color.hpp>
+#include <sstream>
+
+// String conversions for Catch framework
+namespace Catch
+{
+    std::string toString(const sf::String& string)
+    {
+        return string.toAnsiString();
```

Comment:
Why not convert to Utf8? This is probably what will work on most platforms with non-ASCII strings
Now sure how to it handles Windows, though...

Reply:
Because catch expects a `std::string` as return. Not sure if it can deal with `std::basic_string<Uint8>` in all cases. In theory it should work, as `sf::Uint8` is defined as `char` and `std::string` is defined as `std::basic_string<char>`.

Reply:
We can make a string out of sf::basic_string<Uint8> explicitly. That's what a lot of programs do now - they store UTF-8 in std::string. :)

Reply:
Any example code?